src/NodeServices/MessageEvents.cs

```diff
@@ -12,6 +12,19 @@ public LogWarningMessageEventArgs(string msg)
         }
     }
 
+    /// <summary>
+    /// EventArgs for LogInfoMessageEventHandler
+    /// </summary>
+    public class LogInfoMessageEventArgs : EventArgs
+    {
+        public string message { get; internal set; }
+
+        public LogInfoMessageEventArgs(string msg)
+        {
+            message = msg;
+        }
+    }
```

Comment:
We can simply use `LogWarningMessageEventArgs`. For all practical purposes, this is the same as that class so this seems redundant.

Reply:
Will do if you prefer, originally I was going for more separate API.

Reply:
Done